tests/modules/asset_groups/resources/utils.py

```diff
@@ -704,29 +704,56 @@ def patch_asset_group_sighting_as_sighting(
 def read_asset_group_sighting(
     flask_app_client, user, asset_group_sighting_guid, expected_status_code=200
 ):
-    return test_utils.get_dict_via_flask(
+    response = test_utils.get_dict_via_flask(
         flask_app_client,
         user,
         scopes='asset_group_sightings:read',
         path=f'{PATH}sighting/{asset_group_sighting_guid}',
         expected_status_code=expected_status_code,
-        response_200={'guid', 'stage', 'config', 'completion', 'assets'},
+        response_200={
+            'guid',
+            'stage',
+            'config',
+            'completion',
+            'assets',
+            'creator',
+            'asset_group_guid',
+            'sighting_guid',
+        },
     )
+    # so we can capture the example if we want, via -s flag on pytest
```

Comment:
Stole this idea from one of the sighting tests. It's useful to have them always printed out with the -s flag but do you think this is a parrern we should be following for every read util in all the modules?

Reply:
Probably a useful thing to do as API's stop changing so quickly.